src/Command.ts

```diff
@@ -71,6 +71,11 @@ export default abstract class Command {
     }
   }
 
+  protected showWarning(logger: Logger, warning: string): void {
```

Comment:
Let's name this function `warn` as we discussed in the spec

Reply:
Yes, I wasn't sure where to place it. The error function is in the `cli` class, but is implemented different, with console.log. So I placed this in the `Command` file, which is why I aligned the naming with `showDeprecation`. I'll change it.